File: bakalarkaBE/bakalarkaBE/Models/Zaznamyzdielanie.cs
using System;
using System.Collections.Generic;

namespace bakalarkaBE.Models;

public partial class Zaznamyzdielanie
{
    public string Zdielajuci { get; set; } = null!;

    public string Cielovy { get; set; } = null!;

    public int Idzaznamu { get; set; }

    public DateOnly DatumdoZdielanie { get; set; }

    public virtual Doktor CielovyNavigation { get; set; } = null!;

    public virtual Zaznam IdzaznamuNavigation { get; set; } = null!;

    public virtual Doktor ZdielajuciNavigation { get; set; } = null!;
}

File: FE/src/app/models/ZaznamyZdielanie.ts
import {Zaznam} from "./Zaznam";
import {Doktor} from "./Doktor";

export class ZaznamyZdielanie {
  zdielajuci:string = "";
  cielovy:string = "";
  idzaznamu:number = 0;
  datumdoZdielanie:string = "";
  idzaznamuNavigation:Zaznam = new Zaznam();
  zdielajuciNavigation?:Doktor;
  cielovyNavigation?:Doktor;
}
